src/utils/trading/__tests__/symbol.utils.test.ts

```diff
@@ -1,9 +1,15 @@
+import { getSpotQuote, getSpotSymbol } from '../symbol.utils';
+
 describe('Symbol utils', () => {
   describe('getSpotSymbol', () => {
```

Comment:
Is it worth writing sad path tests? (e.g. X//Y, X/Y/Z, /Y, X/) This would mean error handling in the utils.
I'm not yet familiar with where this would be coming from to have an opinion whether this makes sense.